ui/imports/Constants.qml

```diff
@@ -119,4 +119,10 @@ QtObject {
     readonly property int browserEthereumExplorerBlockchair: 3
 
     readonly property int repeatHeaderInterval: 2
+
+    readonly property int maxUploadFiles: 5
+    readonly property var maxUploadFilesizeMB: 0.5
```

Comment:
Should this be `float` instead?

Reply:
[`float` is not supported, but `double` is](https://doc.qt.io/qt-5/qtqml-typesystem-basictypes.html#supported-basic-types) so i've changed it to `double`.